### ISSUE TITLE
Onboarding script/checklist

### ISSUE THREAD

**Issue by knod** [codeforboston/clean-slate#106]
From #65, create wiki page with guidance materials for whoever needs to onboard new members. Link to it in meeting documentation. It's a wiki page because it's for the whole team.

**Comment by knod** [comment]
First draft! See [onboarding checklist](https://github.com/codeforboston/clean-slate/wiki/Onboarding-Checklist).

Dose this close the issue/task? Or does it need to be reviewed first?

**Comment by jeremylang** [comment]
@knod looks good to me based on my understanding of the process and challenges. I added links to documentation per the note on the checklist.

Only thing I'd propose adding is a 7.iiii to ask what amount of time each person is thinking they can contribute. Gets them thinking a bit about how serious they are and lets us know for better planning. (Haven't done one of these, so not sure if that's a "too soon").

**Comment by knod** [comment]
No, I like that idea! I also think I'll add that the group make a practice fist-to-five proposal and decision.

**Comment by knod** [comment]
I've changed 7.iv and made a new 10. Thoughts?

**Comment by jeremylang** [comment]
@knod solid! I like the addition of the fist-to-five. A practice run would definitely help before being in a meeting.

That's all I've got. Looks good.
